spec/openapi.yaml

```diff
@@ -3643,6 +3657,9 @@ paths:
           "template": {
             "id":"$mtId",
             "status":"$statusCode"}
+          "read_timeout": 5000,
+          "retries": 3,
+          "retry_delay": 30
```

Comment:
Can you add these properties to the sections below? Around line 3680

Reply:
the section around 3680 is about type of events , in the other webhook documents it is not listed there

Reply:
Sorry, I mean the section before that. It should be added in the

> A valid webhook must consist of the following properties